bodhi/tests/client/test___init__.py

```diff
@@ -456,10 +456,10 @@ def test_url_flag(self, send_request):
                 mock.MagicMock(return_value='a_csrf_token'))
     @mock.patch('bodhi.client.bindings.BodhiClient.send_request',
                 return_value=client_test_data.EXAMPLE_UPDATE_MUNCH, autospec=True)
-    @mock.patch('__builtin__.raw_input', create=True)
-    def test_query_mine_flag_username_unset(self, mock_raw_input, send_request):
-        """Assert that we use init_username if --user is not given."""
-        mock_raw_input.return_value = 'dudemcpants'
+    @mock.patch('bodhi.client.bindings.input', create=True)
+    def test_query_mine_flag_username_unset(self, mock_input, send_request):
+        """Assert that we use init_username if USERNAME is not set"""
```

Comment:
This comment might be the result of a merge conflict - it should still say the former string:

```
"""Assert that we use init_username if --user is not given."""
```

Bodhi no longer uses the ```USERNAME``` env var.

Reply:
Yes, damn merge conflict. Fixed.